### PR TITLE
half of the design of form and li

### PR DESCRIPTION
**Pull request by sara219**

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -15,4 +16,87 @@
...
+#contact input[type="tel"],
+#contact input[type="url"],
+#contact textarea,
+#contact button[type="submit"] {
+    font: 400 12px/16px "Roboto", Helvetica, Arial, sans-serif;
+}
+
+#contact {
+    background: #F9F9F9;
+    padding: 25px;
```
Comment (mhmdtshref) [inline public/style.css]: Preferred to use percentage `%` for setting `CSS padding`.

File: public/style.css [hunk 2]
```diff
@@ -15,4 +16,87 @@
...
+}
+
+#contact h3 {
+    display: block;
+    font-size: 30px;
+    font-weight: 300;
+    margin-bottom: 10px;
+}
+
+#contact h4 {
```
Comment (mhmdtshref) [inline public/style.css]: I cannot see any `h4` elements inside `#contant` form. This can be deleted

File: public/style.css [hunk 3]
```diff
@@ -15,4 +16,87 @@
...
+    margin: 5px 0 15px;
+    display: block;
+    font-size: 13px;
+    font-weight: 400;
+}
+
+fieldset {
```
Comment (mhmdtshref) [inline public/style.css]: fieldset will be removed (view my comment in the HTML file)

File: public/index.html [hunk 4]
```diff
@@ -23,6 +23,28 @@
             <ul class="traditional" id="traditional"></ul>
         </div>
     </div>
+
+    <div class="container">
+        <form id="contact" action="" method="post">
+            <h3>Add Restaurant</h3>
+            <fieldset>
```
Comment (mhmdtshref) [inline public/index.html]: You don't need to use `fieldset` for each input.
Actually, you don't need to use it at all in this case. Delete it and make sure that you fix the CSS issue after deleting it.

File: public/style.css [hunk 5]
```diff
@@ -15,4 +16,87 @@
...
+    max-width: 400px;
+    width: 100%;
+    margin: 0 auto;
+    position: relative;
+}
+
+#contact input[type="text"],
+#contact input[type="email"],
+#contact input[type="tel"],
+#contact input[type="url"],
```
Comment (mhmdtshref) [inline public/style.css]: The previous 4-lines can be replaced with `#contact input`.

File: public/style.css [hunk 6]
```diff
@@ -15,4 +16,87 @@
...
+#contact input[type="tel"],
+#contact input[type="url"] {
+    width: 100%;
+    border: 1px solid #ccc;
+    background: #FFF;
+    margin: 0 0 5px;
+    padding: 10px;
+}
+
+#contact button[type="submit"] {
```
Comment (mhmdtshref) [inline public/style.css]: You don't need to add `[type="submit"]` to the code, try to minimize the code as possible.

File: public/style.css [hunk 7]
```diff
@@ -15,4 +16,87 @@
...
+    border: none;
+    background: #4CAF50;
+    color: #FFF;
+    margin: 0 0 5px;
+    padding: 10px;
+    font-size: 15px;
+}
+
+
+/* li {
```
Comment (mhmdtshref) [inline public/style.css]: Remove this comment if you don't need it.

File: public/style.css [hunk 8]
```diff
@@ -15,4 +16,87 @@
     width: 30%;
     border: 1px solid black;
     border-radius: 5%;
+}
+
+.container {
+    max-width: 400px;
```
Comment (mhmdtshref) [inline public/style.css]: If you need to add max-width, I think it should be using percentage `%`

**Review by mhmdtshref — CHANGES_REQUESTED** [review]
Great, I've left some comments, check it and I'll review it again.

**Review by mhmdtshref — CHANGES_REQUESTED** [review]
Great, I've left some comments, check it and I'll review it again.

**Review by Nadeen-Sh — APPROVED** [review]

**Pull request merged**